README.md

```diff
@@ -1,12 +1,14 @@
 # cheri-examples
-cheri-riscv sample c programs
+cheri sample C programs for RISC-V and Morello
```

Comment:
Let's line wrap to the same width as everything else in the file.

Reply:
sorry. Done!